src/wasm/wasm.cpp

```diff
@@ -88,6 +88,8 @@ Name EXIT("exit");
 Name SHARED("shared");
 Name EVENT("event");
 Name ATTR("attr");
+Name WASI("wasi");
+Name WASI_UNSTABLE("wasi_unstable");
```

Comment:
It premature to add WASI here, unless I'm missing something.  The plan is not to go from wasi_unstable to just wasi, but rather split it into modules.

Reply:
Oh, I didn't know that, will fix.